### PR TITLE
Cleanup ActionMap.py

### PR DESCRIPTION
**Pull request by persianpros**

### CODE REVIEW

**Comment by littlesat** [comment]
Except for one line not a real improvement

File: lib/python/Components/ActionMap.py [hunk 1]
```diff
@@ -57,7 +59,7 @@ def destroy(self):
 class NumberActionMap(ActionMap):
 	def action(self, contexts, action):
 		numbers = ("0", "1", "2", "3", "4", "5", "6", "7", "8", "9")
-		if (action in numbers and self.actions.has_key(action)):
+		if action in numbers and self.actions.has_key(action):
```
Comment (Taapat) [inline lib/python/Components/ActionMap.py]: Please change also has_key to in: http://legacy.python.org/dev/peps/pep-0290/#testing-dictionary-membership

**Comment by littlesat** [comment]
You mean 

If action in numbers and action in actions?

**Comment by Taapat** [comment]
I mean:
`If action in numbers and action in self.actions:`

**Comment by littlesat** [comment]
;)

**Comment by Taapat** [comment]
And if we take a look deeper, I think that we can also remove variable numbers:
`If action in ("0", "1", "2", "3", "4", "5", "6", "7", "8", "9") and action in self.actions:`

**Comment by littlesat** [comment]
I agree...

**Comment by littlesat** [comment]
Can we have a new merge,requesr?

**Pull request closed without merging**